.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,12 @@
 # Please read first!
 Please use [discuss-webrtc](https://groups.google.com/forum/#!forum/discuss-webrtc) for general technical discussions and questions.
+If you have found an issue/bug with the native `libwebrtc` SDK please create an issue at [bugs.webrtc.org](https://bugs.chromium.org/p/webrtc/issues/list)
```

Comment:
The canonical link is https://bugs.webrtc.org/ - which redirects to this URL, but we might as well give the canonical one.